requirements.txt

```diff
@@ -15,4 +15,4 @@ sdx-common==0.7.0
 six==1.10.0
 structlog==16.1.0
 Werkzeug==0.11.9
-sdc-cryptography==0.1.6
\ No newline at end of file
+sdc-cryptography==0.2.0-RC1
```

Comment:
Probably shouldn't be taking branches back to master that rely on release candidates - we certainly shouldn't release anything to prod that does.